keyboards/latinpad/readme.md

```diff
@@ -0,0 +1,12 @@
+# Latinpad
+
+macro 2 rotary encoder pad
+
+* Keyboard Maintainer: Latincompass
+* Hardware Availability: https://github.com/latincompass/qmk_firmware/tree/master/keyboards/latinpad
```

Comment:
Still no hardware address.

Reply:
Sir ，I think: You can give me an example of the process of error correction, so as to save your precious work.Thanks a lot